chef/cookbooks/pacemaker/attributes/default.rb

```diff
@@ -15,15 +15,19 @@
 
 case node.platform
 when 'suse'
-  default[:pacemaker][:platform][:packages] = %w(pacemaker crmsh)
+  # Install xorg-x11-xauth by default so that 'crm configure graph'
+  # (which uses dotty) will show a nice configuration graph over ssh
+  # with X11 forwarding.
+  default[:pacemaker][:platform][:packages] = %w(
+    pacemaker crmsh xorg-x11-xauth
```

Comment:
Well, it's kind of the same reason why we never installed hb_gui by default, I'd say.

With the move to SLE12, hb_gui being gone, we'd want to either get rid of the magic option or update the hint like you mention. So I'd be in favor of that.